.changeset/red-llamas-spit.md

```diff
@@ -1,8 +1,8 @@
 ---
-"@fluidframework/container-utils": major
+"@fluidframework/telemetry-utils": major
```

Comment:
?

Reply:
You can't have a changeset that refers to a package that doesn't exist.

Reply:
Since most of the functions went to telemetry-tuils, I used that package.

Reply:
Hmm leaves me wondering how do we announce "this package is now completely gone and there are no replacements for its exports" if we ever need to. And in fact I think it kind of applies here? The things that moved over, I imagine might not be intended for use by external consumers, and this makes it feel like if they were using them, they should now just use them from telemetry-utils. Disclaimer: I didn't go check _what_ moved, so might be wrong on that.

Reply:
> leaves me wondering how do we announce "this package is now completely gone and there are no replacements for its exports" if we ever need to

I'm not worried about that. In that case we don't really need to announce the package is gone. The last version of the package published will have all the APIs deprecated (e.g. we just did this with common-utils), and once deleted, there's just never a new version published. That's basically what happened here - we shipped deprecations in 6.2.0 (I assume we announced the deprecations in the release notes then), and that's the "final" version of that package.

In other words, the decision of replacement APIs and announcing them really needs to be made when deprecating them, not when removing them.

Reply:
Maybe put in the body another sentence or phrase "Many of them were moved to telemetry-utils" or whatever

Reply:
I think maybe you guys are missing the whole changeset content, which is:

> All members of the **@fluidframework/container-utils** package have been deprecated and the package is now removed.
> 
> Migration by API member:
> -   `ClientSessionExpiredError` (deprecated in `2.0.0-internal.6.2.0`): No replacement API offered.
> -   `DataCorruptionError` (deprecated in `2.0.0-internal.6.2.0`): Import from [@fluidframework/telemetry-utils](https://www.npmjs.com/package/@fluidframework/telemetry-utils) instead.
> -   `DataProcessingError` (deprecated in `2.0.0-internal.6.2.0`): Import from [@fluidframework/telemetry-utils](https://www.npmjs.com/package/@fluidframework/telemetry-utils) instead.
> -   `DeltaManagerProxyBase` (deprecated in `2.0.0-internal.6.1.0`): No replacement API offered.
> -   `extractSafePropertiesFromMessage` (deprecated in `2.0.0-internal.6.2.0`): Import from [@fluidframework/telemetry-utils](https://www.npmjs.com/package/@fluidframework/telemetry-utils) instead.
> -   `GenericError` (deprecated in `2.0.0-internal.6.2.0`): Import from [@fluidframework/telemetry-utils](https://www.npmjs.com/package/@fluidframework/telemetry-utils) instead.
> -   `ThrottlingWarning` (deprecated in `2.0.0-internal.6.2.0`): No replacement API offered.
> -   `UsageError` (deprecated in `2.0.0-internal.6.2.0`): Import from [@fluidframework/telemetry-utils](https://www.npmjs.com/package/@fluidframework/telemetry-utils) instead.

I think that's pretty thorough.